addon/components/frost-date-picker.js

```diff
@@ -50,7 +50,8 @@ export default Component.extend(EventsProxyMixin, {
   @readOnly
   @computed('value')
   _value (value) {
-    if (value === undefined) {
+    const emptyValues = new Set(['', undefined, null])
+    if (value in emptyValues) {
```

Comment:
Why not use `isEmpty()`. https://www.emberjs.com/api/ember/2.12/namespaces/Ember/methods/isEmpty?anchor=isEmpty

Reply:
I originally was just checking for falsey values, but @gknoy asked to see the empty values explicitly. I'm fine with using `isEmpty()` if that is explicit enough for him.